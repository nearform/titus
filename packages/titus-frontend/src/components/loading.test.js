import React from "react";
import { render, cleanup } from "@testing-library/react";

import Loading from './loading'

describe('<Loading />', () => {

  afterEach(cleanup);

  it("renders", () => {
    const { asFragment } = render(<Loading />);
    expect(asFragment()).toMatchSnapshot();
  });
})




