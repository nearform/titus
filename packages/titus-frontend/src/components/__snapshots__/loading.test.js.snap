// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> should render correctly 1`] = `
<DocumentFragment>
  <div
    class="loader"
  />
</DocumentFragment>
`;
